tests/MongoDB.Driver.Tests/Specifications/crud/CrudTestRunner.cs

```diff
@@ -106,6 +105,7 @@ private void AssertEventsIfNeeded(EventCapturer eventCapturer, BsonDocument test
             {
                 var actualEvents = eventCapturer.Events;
                 var expectedEvents = expectations.AsBsonArray.Cast<BsonDocument>().ToList();
+                RemoveExtraFindCommandsFromActualEvents(actualEvents, expectedEvents.Count);
```

Comment:
There are two sources of extra Finds:

1. Aggregate with $out/$merge does an automatic Find on the output collection
2. The test runner sometimes reads the test collection to verify its contents

Reply:
The second could be dealt with by using a separate MongoClient (is there a global one lying around somewhere already?) to verify test collection contents.

Reply:
I believe the Java driver handles the first one because a long time ago we added an `AggregateIterable.ToCollection` method that  can be be used when you don't want a result back.  Could add a method like that in .NET if there's a good way to do it. and use it in tests if we detect a $out/$merge. (Goes without saying that it would be under a separate ticket!)